citi-onboarding/virtu-onepage
# Feature/our services integration

### What I did
- Added new route to server in order to connect it with front end
- Integrated the Services cards from OurServices section with data coming from Keystone CMS
- Changed the "Saiba mais" button in the CardService component so that when you click it, you get redirected to the Contact section

### How to test
1. Enter server folder with `cd server`
2. Execute the command `yarn` to download the server modules
3. Execute the command `yarn start` to start the server
4. Execute the command `cd ../client` and then `yarn` to download the client modules
5. Execute the command `yarn start` to start the client
6. Open your browser on `http://localhost:3001/admin` to view keystone admin
7. Log in with the credentials available at Notion
8. Open another browser page on `http://localhost:3000` to see the client page
9. Compare the information's passed in the "Servicos" section of Keystone with the ones seen in the OurServices section from the client. You can also try to change it to check if it works

Resolves #32
